src/Symfony/Component/Routing/Tests/Matcher/UrlMatcherTest.php

```diff
@@ -22,6 +23,24 @@
 
 class UrlMatcherTest extends TestCase
 {
+    public function testContext()
+    {
+        $matcher = $this->getUrlMatcher(new RouteCollection());
+
+        $matcher->setContext($requestContext = new RequestContext('bar'));
+        $this->assertSame($requestContext, $matcher->getContext());
+    }
+
+    public function testMatchRequest()
```

Comment:
Where's the additional value in this test? Doesn't `testPatternMatchAndParameterReturn()` already cover everything that is tested here?